pkg/api/org_users.go

```diff
@@ -211,15 +212,31 @@ func (hs *HTTPServer) getOrgUsersHelper(c *models.ReqContext, query *org.GetOrgU
 
 	filteredUsers := make([]*org.OrgUserDTO, 0, len(result))
 	userIDs := map[string]bool{}
+	authLabelsUserIDs := make([]int64, 0, len(result))
 	for _, user := range result {
 		if dtos.IsHiddenUser(user.Login, signedInUser, hs.Cfg) {
 			continue
 		}
 		user.AvatarURL = dtos.GetGravatarUrl(user.Email)
 
 		userIDs[fmt.Sprint(user.UserID)] = true
+		authLabelsUserIDs = append(authLabelsUserIDs, user.UserID)
 		filteredUsers = append(filteredUsers, user)
 	}
+	ccc := c.Req.Context()
+	modules, err := hs.authInfoService.GetUserLabels(ccc, models.GetUserLabelsQuery{
+		UserIDs: authLabelsUserIDs,
+	})
+
+	if err != nil {
+		hs.log.Warn("failed to retrieve users IDP label", err)
+	}
+
+	for i := range filteredUsers {
```

Comment:
Is it possible to combine this with the loop below?

Reply:
Changed as requested